app/payment-link-v2/amount/amount.njk

```diff
@@ -0,0 +1,52 @@
+{% extends "../../views/layout.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% block pageTitle %}
+  {{ __p('paymentLinksV2.amount.enterAmountToPay') }} - {{ serviceName }}
+{% endblock %}
+
+{% block contentBody %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    {{ govukBackLink({
+      text: "Back",
+      href: backLinkHref
+    }) }}
+
+    <form action="/pay/{{ productExternalId }}/amount" method="post" name="startPaymentForm" class="push-bottom">
+      <input id="csrf" name="csrfToken" type="hidden" value="{{ csrf }}"/>
+      <div class="currency-input govuk-form-group {% if flash.genericError %}govuk-form-group--error{% endif %}">
```

Comment:
We'll replace this stuff to do with displaying the error message here, as we'll be switching to ensure we use the design system error summary component and use the `errorMessage` field on the `govukInput` to display the inline error message.

We'll also stop using flash to pass those errors to the form.

It's fine to replace this in the PR that does the error handling for the POST controller though.

Reply:
We will fix this in another PR - thanks